book/src/snapshot-restore-feature.md

```diff
@@ -55,15 +55,21 @@ The snapshot controller is deployed by the Kubernetes distributions and is respo
 
 The CSI external-snapshotter sidecar watches Kubernetes VolumeSnapshotContent CRD objects and triggers CreateSnapshot/DeleteSnapshot against a CSI endpoint.
 
+### Snapshot Validation Webhook
+
+There is a new validating webhook server which provides tightened validation on snapshot objects. This SHOULD be installed by all users of this feature. See [Snapshot Validation Webhook](snapshot-validation-webhook.md) for more details on how to use the webhook.
```

Comment:
clarify here that this should be installed by the Kubernetes distros along with the snapshot-controller; not end users

Reply:
done